data.sql

```diff
@@ -4,3 +4,9 @@ insert into animals values (default, 'Agumon', '20200206', 1, true, 14.23);
 insert into animals values (default, 'Gabumon', '20181115', 2, true, 8.00);
 insert into animals values (default, 'Pikachu', '20210107', 1, false, 15.04);
 insert into animals values (default, 'Devimon', '20170512', 5, true, 11.00);
+insert into animals values (default, 'Charmander', '20200208', 0, false, 11.00);
+insert into animals values (default, 'Plantmon', '20221115', 2, true, 5.7);
+insert into animals values (default, 'Squirtle', '19930402', 3, false, 12.13);
```

Comment:
- [ ] Let's update weight_kg from 12.13 to -12.13 as given in the guidelines.

Reply:
Done ✔